client.py

```diff
@@ -72,6 +73,138 @@ def sql_example(hz_client):
         exit(0)
 
 
+def json_serialization_example(hz_client):
+    create_mapping_for_countries(hz_client)
+
+    populate_countries_map(hz_client)
+
+    select_all_countries(hz_client)
+
+    create_mapping_for_cities(hz_client)
+
+    populate_city_map(hz_client)
+
+    select_cities_by_country(hz_client, "AU")
+
+    select_countries_and_cities(hz_client)
+    exit(0)
```

Comment:
Minor: It is okay to remove this from the Python code samples, as after this function is called, we will shut down the client and the process will automatically exit, unlike the Node.js client

Reply:
Just checked, it works, thanks. Removed: https://github.com/hazelcast/hazelcast-cloud-python-sample-client/pull/7/commits/c34c8cbc006c44c1be5f15b763b34f152e54d17d